RELEASE-NOTES.txt

```diff
@@ -14,6 +14,7 @@
 * [***] Block Editor: Audio block now available on WP.com sites on the free plan. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3523]
 * [**] You can now create a Site Icon for your site using an emoji. [#16670] 
 * [*] Fix notice overlapping the ActionSheet that displays the More Actions in the Editor. [#16658]
+* [*] The quick action buttons will be hidden when iOS is using a accessibility font sizes. [#16701]
```

Comment:
This noted didn't make it into the editorialized release notes. Spoke with @diegoreymendez, the author, offline and he said it's a minor change so that's okay.